Dockerfile

```
@@ -54,16 +54,17 @@ RUN \
 # Copy all of the files into the source directory and then switch contexts
 COPY . ${TNF_SRC_DIR}
 WORKDIR ${TNF_SRC_DIR}
-RUN make install-tools && make build-cnf-tests
+RUN \
+	make install-tools \
+	&& make classification-js \
+	&& make build-cnf-tests
```

```suggestion
RUN make build-cnf-tests classification-js install-tools
```

Make works with multiple arguments, put the argument in alphabetical order.

---

Fixed in https://github.com/test-network-function/cnf-certification-test/pull/1200/commits/b173b199ea7787ccd16153c2d3e151f690156035
install-tools should run before build-cnf-tests